### PR TITLE
Make test matrix automatically

### PR DESCRIPTION
**Pull request by hackallcode**
Before the patch, it was possible forget to add molecule test to matrix.

Now matrix for molecule tests with defaults params is calculated automatically.

### CODE REVIEW

File: .github/helpers/count_molecule_matrix.py [hunk 1]
```diff
@@ -2,39 +2,93 @@
...
+
+MOLECULE_SCENARIOS_PATH = './molecule'
+DEFAULT_SCENARIO = 'default'
+IGNORED_PATHS = [
+    'common',
+]
+
+DEFAULT_MOLECULE_COMMAND = 'test'
+NOT_DEFAULT_MOLECULE_COMMANDS = [
+    # TODO: Uncomment after fixing the check mode
```
Comment (dokshina) [inline .github/helpers/count_molecule_matrix.py:19]: Is there an issue to fix check mode?

File: .github/helpers/count_molecule_matrix.py [hunk 2]
```diff
@@ -48,38 +102,30 @@ def main(event_name, repo_owner, review_state, ref):
...
-        tdg_matrix.append(get_tdg_params(molecule_scenario='config_upload_tdg', tdg_version='1.7.1-0-g92920bea'))
-
-        # TDG functionality: common.app_version [+]; admin.upload_config_api [+];
-        tdg_matrix.append(get_tdg_params(molecule_scenario='config_upload_tdg', tdg_version='1.6.16-0-g7e140c94'))
-        tdg_matrix.append(get_tdg_params(molecule_scenario='config_upload_tdg', tdg_version='1.7.7-0-g76c31fca'))
+        ee_matrix.append(get_ee_params())
+
+        for name in filter(
+            lambda scenario:
+            scenario != DEFAULT_SCENARIO and scenario not in IGNORED_PATHS and scenario not in TDG_SCENARIOS,
```
Comment (dokshina) [inline .github/helpers/count_molecule_matrix.py]: What about use the non-lambda function here? Just to make code more readable. Or simple add `DEFAULT_SCENARIO, TDG_SCENARIOS` to `IGNORED_PATHS`?

File: .github/helpers/count_molecule_matrix.py (hunk 1 above)
Comment (hackallcode) [reply .github/helpers/count_molecule_matrix.py:19]: No

Comment (hackallcode) [reply .github/helpers/count_molecule_matrix.py:19]: Yes
https://github.com/tarantool/ansible-cartridge/issues/360

**Review by dokshina — APPROVED** [review]

**Pull request merged**